Packs/AzureActiveDirectory/Integrations/AzureADIdentityProtection/AzureADIdentityProtection.yml

```diff
@@ -425,9 +459,137 @@ script:
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Risk level of the detected risky user. The possible values are low,
+        medium, high, hidden, none, and unknownFutureValue.
+      isArray: false
+      name: risk_level
+      predefined:
+      - low
+      - medium
+      - high
+      - hidden
+      - none
+      - unknownFeatureValue
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: State of the user's risk. The possible values are none, confirmedSafe,
+        remediated, dismissed, atRisk, confirmedCompromised, and unknownFutureValue.
+      isArray: false
+      name: risk_state
+      predefined:
+      - none
+      - confirmedSafe
+      - remediated
+      - dismissed
+      - atRisk
+      - confirmedCompromised
+      - unknownFutureValue
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Details of the detected risk. The possible values are limited to none, adminGeneratedTemporaryPassword,
+        userPerformedSecuredPasswordChange, userPerformedSecuredPasswordReset, adminConfirmedSigninSafe,
+        aiConfirmedSigninSafe, userPassedMFADrivenByRiskBasedPolicy, adminDismissedAllRiskForUser,
+        adminConfirmedSigninCompromised, hidden, adminConfirmedUserCompromised, and unknownFutureValue.
+      isArray: false
+      name: risk_detail
+      predefined:
+      - none
+      - adminGeneratedTemporaryPassword
+      - userPerformedSecuredPasswordChange
+      - userPerformedSecuredPasswordReset
+      - adminConfirmedSigninSafe
+      - aiConfirmedSigninSafe
+      - userPassedMFADrivenByRiskBasedPolicy
+      - adminDismissedAllRiskForUser
+      - adminConfirmedSigninCompromised
+      - hidden
+      - adminConfirmedUserCompromised
+      - unknownFutureValue
+      required: false
+      secret: false
+    - default: false
+      description: A custom query in OData syntax. Using this overrides all
+        arguments, except for next_link. For more details, see https://docs.microsoft.com/en-us/graph/query-parameters.
+      isArray: false
+      name: filter_expression
+      required: false
+      secret: false
+    - default: false
+      defaultValue: '50'
+      description: Number of results to provide.
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    - default: false
+      description: A link that specifies a starting point for subsequent calls.
+        Using this argument overrides all other arguments.
+      isArray: false
+      name: next_link
+      required: false
+      secret: false
+    - default: false
+      description: Risky user principal name.
+      isArray: false
+      name: user_name
+      required: false
+      secret: false
+    deprecated: false
+    description: Retrieves the properties of a collection of riskDetection objects.
+    execution: false
+    name: azure-ad-identity-protection-fetch
```

Comment:
What is this command?

Reply:
Unused. Removed. https://github.com/demisto/content/pull/14352/commits/dee9a671ecaf85ec965f48a8a3031a92c30f4634